src/commands.js

```diff
@@ -47,6 +47,38 @@ module.exports = {
 			route.send('Error learning that fact: ' + err);
 		});
 	},
+	say : function (route, message) {
+		
+		var bot = this;
+		var output = message.join(' ');
+
+		console.log('Forced to say fact:', output);
+
+		// Replace who with the triggerer
```

Comment:
Can you try and extract the replacement code from here and the real fact listener into one function to reduce code duplication?